lib/event_controller.rb

```diff
@@ -50,25 +51,10 @@ def verify_url(data)
   end
 
   def handle_event(data)
-    return unless event_callback?(data)
+    checker = SlackApi::Checker.new(data)
+    return unless checker.require_answer?
```

Comment:
with `Request` rather than `Checker` we would be able to write:

```
request = SlackApi::Request.new(data)
return unless request.require_answer?
```

which reads nicely I think. :)

What do you think?

Reply:
Love it 👍